content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,190 @@
+
+Configure physical warehouse locations
+======================================
+
+Navigate back to the :guilabel:`Locations` overview (via the breadcrumbs), and remove any filters
+in the :guilabel:`Search Bar`. Then, click into the first physical warehouse :guilabel:`Location`
+that was previously created to be a *child location*, and click :guilabel:`Edit`.
+
+Under :guilabel:`Parent Location`, select the virtual :guilabel:`Warehouse` from the drop-down
+menu, and :guilabel:`Save` changes. Then, navigate back to the :guilabel:`Locations` overview, and
+repeat this step for the second physical :guilabel:`Warehouse` stock :guilabel:`Location`. Be sure
+to :guilabel:`Save` changes again.
 
-Sell a product from the virtual warehouse
-=========================================
+Both locations are now *child locations* of the virtual warehouse *parent location*. This allows
+stock to be taken from multiple locations to fulfill a single sales order, if there is not enough
+stock in any one location (provided they are both tied to the same virtual warehouse *parent
+location*).
 
-Let’s say you have two products, one stored in warehouse A and one stored in warehouse B. Now, you
-can create a new quotation for one of each product. Go to other information and choose Warehouse A+B
-in the shipping information.
+Example flow: Sell a product from a virtual warehouse
+=====================================================
 
-.. image:: stock_warehouses/ware3.png
+To sell products from multiple warehouses using a virtual *parent* location, there must be at least
+**two** products and at least **two** warehouses configured - with at least **one** product with
+quantity on-hand in each warehouse, respectively.
+
+To create a new Request for Quotation, or RFQ, navigate to the :menuselection:`Sales` app, and
+click :guilabel:`Create` from the :guilabel:`Quotations` overview. Fill out the information on the
+new quotation by adding a :guilabel:`Customer`, and click :guilabel:`Add a product` to add the two
+products stored in the two warehouses.
+
+Then, click the :guilabel:`Other Info` tab. Under the :guilabel:`Delivery` section, change the
+:guilabel:`Warehouse` listed to the virtual warehouse that was previously created. Once the
+warehouse has been changed, click :guilabel:`Confirm` to confirm the quotation and create a sales
+order.
+
+Now that the quotation has been confirmed as a sales order, click the :guilabel:`Delivery` smart
+button. From the :guilabel:`Warehouse Delivery form`, click :guilabel:`Edit`, and change the
+:guilabel:`Source Location` to the *Stock* location of the virtual warehouse that was previously
+created, it it was not already automatically set as the *source location*.
+
+.. important::
+   The :guilabel:`Source Location` and :guilabel:`Warehouse`, under the :guilabel:`Other Info` tab
+   on the :guilabel:`Sales Order form`, must match in order for the products included in the sales
+   order to be pulled from different warehouses.
```

Comment:
This is a very touchy setup if not performed correctly. 
Would include some basic troubleshooting for each possible error/scenario that might show up (as I ran into all 3 😓 )

```suggestion
.. important::
   The :guilabel:`Source Location` on the warehouse delivery form and the :guilabel:`Warehouse`
   under the :guilabel:`Other Info` tab on the sales order form *must* match in order for the
   products included in the sales order to be pulled from different warehouses.

  - If the virtual warehouse is not the value in the :guilabel:`Source Location` field on the
    warehouse delivery form, then click :guilabel:`Edit`, make the change, and click
    :guilabel:`Save`.
  - If the virtual warehouse is not the value in the :guilabel:`Warehouse` field on the sales order,
    then a new quotation may need to be generated.
  - If whe :guilabel:`Warehouse` field is missing on the sales order form, then the virtual
    warehouse (and its children warehouses) may not have been set up correctly, in which case,
    review the documentation above again to make sure all settings/configuration were done properly.
```